spec/file_reference_installer/header_file_reference_installer_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.